{"uuid":"0b60bfd3-f5b6-452b-833e-86016354e36d","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["fd7a817c-6e5b-4141-89c0-08c5fe1d9441"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378299523,"stop":1673378299523}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378299538,"stop":1673378299538},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378299923,"stop":1673378299924},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378300054,"stop":1673378300054},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"1693479d-05e4-4346-8143-71738daff439-attachment","type":"image/png"},{"name":"Myscreenshot","source":"6095e32e-da07-460c-ba3e-654007b103b3-attachment"}],"parameters":[],"start":1673378300074,"stop":1673378300395}],"start":1673378299523,"stop":1673378300396}